src/util/thread.h

```diff
@@ -26,6 +26,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "../threading/mutex_auto_lock.h"
 #include "porting.h"
 #include "log.h"
+#include "container.h" // MutexedQueue
```

Comment:
Well whatever, less includes inside the most common headers is faster to compile so I'll go with that.